code/datums/emotes.dm

```diff
@@ -24,6 +24,7 @@
 	var/sound //Sound to play when emote is called
 	var/vary = FALSE	//used for the honk borg emote
 	var/only_forced_audio = FALSE //can only code call this event instead of the player.
+	var/cooldown = 0.8 SECONDS //austation -- cooldown var for all emotes
```

Comment:
you can modularize this to its own file, since it's just a variable. Just make a file in the austation dir called `code/datums/emotes.dm`, add a `/datum/emote` and add the variable definition to it. You don't need to edit this file. Make sure to add it to includes.dm